app/views/issue_mailer/notify_subscribers.slim

```diff
@@ -12,6 +12,8 @@ body topmargin="0" leftmargin="0" marginheight="0" marginwidth="0" style="-webki
       - @issue.links.each do |link|
         li
           a href="#{link.url}" target="_blank" style="font-size: 18px; color: #4183c4; text-decoration: none;" #{link.title}
+          - if link.time_to_read
+            p Estimated time to read: #{link.time_to_read}
```

Comment:
there will be too many 'estimated time to read ' words in the email. can we render this value after the link text in the same line? maybe in brackets